src/containers/Homepage/index.js

```diff
@@ -22,10 +44,7 @@ class Homepage extends React.Component {
             <EventCard eventData={event1} loading={loading} />
           </Col>
           <Col xs={24} sm={24} md={24} lg={12} xl={12}>
-            <Card title={<h3>Thank you for joining us at our Bangor town hall!</h3>} className="event-success-card" />
```

Comment:
Not now, but it would be nice to factor this into a `Past event` component. That would take a title (like Bangor town hall) as a prop, so we could switch it in more easily.

Reply:
yea